doc-content/enterprise-only/vehicle-route-planning/run-locally-noninteractive-proc.adoc

```diff
@@ -0,0 +1,21 @@
+[id='run-locally-noninteractive-proc_{context}']
+
+= Run the OptaWeb Vehicle Routing runLocally.sh script in non-interactive mode
+
+Use the non-interactive mode to specify an existing region and start OptaWeb Vehicle Routing with a single command.
+This is useful for switching between regions quickly or when doing a demo.
+
+.Prerequisites
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* The OSM file for the region that you want to use has been downloaded. For information about downloading OSM files, see xref:run-locally-interactive-proc_{context}[].
+* Internet access is available.
+
+
+.Procedure
+. Change directory to the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaweb-vehicle-routing/sources` directory.
+. Execute the following command where `<REGION>` is a two letter country code for a region that you previously downloaded. For a list of country codes, see https://www.iso.org/iso-3166-country-codes.html[_ISO 3166 Country Codes_].
```

Comment:
This is confusing. It seems the terms region and country code need a definition.

h3. Region
Region is an arbitrary area on the Earth map represented by an OSM file. It might be a country but also a city, a continent or a group of countries that are frequently used together, for example [DACH](http://download.geofabrik.de/europe/dach.html), which stands for Germany (DE), Austria (AT), and Switzerland (CH).

h3. Country code
Country code is a two-letter code assigned to a country by the [ISO-3166](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) standard. It can be used to filter geosearch results. Since you can work with a region that spans multiple countries (for example the DACH region), the application accepts a list of country codes so that geosearch filtering can be used with such regions.

h3. Geosearch
Is a type of query where you give an address or a [toponymy](https://en.wikipedia.org/wiki/Toponymy) as the search keyword and get a number of GPS locations as a result. The number of results depends on how unique the search keyword is but most toponymies are non-unique. One way of filtering out non-relevant results is only including places in the country your working regions is inside (or multiple countries if it's a larger region).

Reply:
I still need to add these definitions.

Reply:
@yurloc I added these to the first section. Please review.